libbpf-rs/tests/bin/src/kprobe.bpf.c

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+
+SEC("kprobe/bpf_fentry_test1")
+int handle__kprobe(void *ctx)
+{
+    return 0;
+}
```

Comment:
Use tab for indentation. I am preparing a patch to format bpf code.

Reply:
Oops, thanks for noticing.